dan_layer/state_store_sqlite/src/schema.rs

```diff
@@ -224,7 +224,7 @@ diesel::table! {
         evidence -> Text,
         remote_evidence -> Nullable<Text>,
         transaction_fee -> BigInt,
-        leader_fee -> BigInt,
+        leader_fee -> Nullable<Text>,
```

Comment:
I would prefer two separate fields, but this is fine as well

Reply:
Yeah definitely better for queriability. Since there's a conflict I'll make those changes too.